lib/screens/login_page.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:badiup/sign_in.dart';
+import 'package:flutter/material.dart';
+import 'home_page.dart';
+
+class LoginPage extends StatefulWidget {
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Container(
+        color: Colors.white,
+        child: Center(
+          child: Column(
+            mainAxisSize: MainAxisSize.max,
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              Text(
+                'Badi up', 
+                style: TextStyle(fontSize: 30.0),
+              ),
+              SizedBox(height: 50),
+              _loginButton(),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _loginButton() {
```

Comment:
rename method name to _buildLoginButton. General rule of thumb is method name should contain a verb + object name.